apps/test/unit/lib/levelbuilder/AllVocabulariesEditorTest.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import {mount} from 'enzyme';
+import sinon from 'sinon';
+import {expect} from '../../../util/reconfiguredChai';
+import {UnconnectedAllVocabulariesEditor as AllVocabulariesEditor} from '@cdo/apps/lib/levelbuilder/AllVocabulariesEditor';
+
+describe('AllVocabulariesEditor', () => {
+  let defaultProps, addVocabulary, updateVocabulary, removeVocabulary;
+  beforeEach(() => {
+    addVocabulary = sinon.spy();
+    updateVocabulary = sinon.spy();
+    removeVocabulary = sinon.spy();
+    defaultProps = {
+      vocabularies: [
+        {id: 1, key: '1', word: 'word1', definition: 'def1'},
+        {id: 2, key: '2', word: 'word2', definition: 'def2'}
+      ],
+      addVocabulary,
+      updateVocabulary,
+      removeVocabulary,
+      courseVersionId: 2021,
+      courseName: 'test-course-2021'
+    };
+  });
+
+  it('renders default props', () => {
+    const wrapper = mount(<AllVocabulariesEditor {...defaultProps} />);
+    expect(wrapper.find('tr').length).to.equal(3);
+  });
+
+  it('can remove a vocabulary', () => {
```

Comment:
maybe add a test for checking you can add vocabulary?

Reply:
I was struggling on what to actually test for that path as the logic is mostly handled through another component. I'm adding a test that clicking the button actually brings up that component but not sure what else to add.